NOTICE

```diff
@@ -0,0 +1,32 @@
+# This is the official list of rcl_executor copyright holders and authors.
+#
+# Often employers or academic institutions have ownership over code that is
+# written in certain circumstances, so please do due diligence to ensure that
+# you have the right to submit the code.
+#
+# When adding J Random Contributor's name to this file, either J's name on its
+# own or J's name associated with J's organization's name should be added,
+# depending on whether J's employer (or academic institution) has ownership
+# over code that is written for this project.
+#
+# How to add names to this file:
+#     Individual's name <submission email address>.
+#
+# If Individual's organization is copyright holder of her contributions add the
+# organization's name, optionally also the contributor's name:
+#
+#     Organization's name
+#         Individual's name <submission corporate email address>
+#
+# Please keep the list sorted.
+
+Open Source Robotics Foundation, Inc.
+    William Woodall <william@osrfoundation.org>
+    Dirk Thomas <dthomas@osrfoundation.org>
+    Esteve Fernandez <esteve@osrfoundation.org>
+
+Robert Bosch GmbH
+    Jan Staschulat <jan.staschulat@de.bosch.com>
+    Ralph Lange <ralph.lange@de.bosch.com>
+
+Loïc Dauphin <astralien3000@yahoo.fr>
```

Comment:
missing newline

Reply:
Fixed in 54f0bbc54a33cc601764ab1d53fe3119d9e61c8b